source/util.js

```diff
@@ -61,3 +62,8 @@ exports.getTagVersionPrefix = pMemoize(async options => {
 		return 'v';
 	}
 });
+
+exports.getRootDir = () => {
```

Comment:
This method is redundant as you could get the root directory of the package by using [`pkg-dir`](https://github.com/sindresorhus/pkg-dir) (see [`index.js`](https://github.com/sindresorhus/np/blob/3925527073c33554763b48b05947852dc57d6c22/source/index.js#L59)).

Reply:
Nice! Updated.